ingenialink/dictionary.py

```diff
@@ -114,6 +114,24 @@ def labels(self, cat_id: str) -> Dict[str, str]:
         return self._categories[cat_id]
 
 
+@dataclass
+class DictionaryError:
+    id: str
```

Comment:
Why don't you store the parsed int also here?

Reply:
To keep the format as it was before. If we store an int, we will have to make changes in ML3.

Reply:
I would expect the GUI to do the formatting of values so the backwards compatibility only on the __iter__ method, not the new API